example/src/main/scala/cilib/example/CSVExample.scala

```diff
@@ -0,0 +1,147 @@
+package cilib
+package example
+import java.io.File
+
+import cilib.benchmarks.Benchmarks
+import cilib.exec.Runner._
+import cilib.exec._
+import cilib.io._
+import cilib.pso.Defaults._
+import cilib.pso._
+import eu.timepit.refined.auto._
+import scalaz.Scalaz._
+import scalaz._
+import scalaz.concurrent.Task
+import scalaz.effect.IO.putStrLn
+import scalaz.effect._
+import scalaz.stream._
+import spire.implicits._
+import spire.math.Interval
+
+object CSVExample extends SafeApp {
```

Comment:
We should perhaps make this more generic (the name of the object). The reason being that the only difference between this example and the one that uses the parquet output is the sink used on line 135.